add a new target BLADE_F4

hakrc1 opened the pull request:
add a new target BLADE_F4

hakrc1 committed:
add a new target BLADE_F4

hakrc1 committed:
add new target BLADE_PRO_H7

hakrc1 committed:
deal serial conut

mmosca commented:
Hi @hakrc1, good to see you want to submit a target.

Are you in contact with one of the developers already?

According to the hardware policy, we need at least 2 members of the dev team to receive samples for validation, before we can merge this as an official target.

Here is the link for the policy, in case you would like to have a look:
https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md

If you are not in touch with any devs, let me know and I can try to organize get you in touch with the right people.

HAKRC commented:
> Hi @hakrc1, good to see you want to submit a target.您好，很高兴看到您要提交目标。
> 
> Are you in contact with one of the developers already?您是否已经与其中一位开发人员取得了联系？
> 
> According to the hardware policy, we need at least 2 members of the dev team to receive samples for validation, before we can merge this as an official target.根据硬件策略，我们至少需要 2 名开发团队成员接收样本进行验证，然后才能将其合并为正式目标。
> 
> Here is the link for the policy, in case you would like to have a look:以下是该政策的链接，如果您想查看： https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md
> 
> If you are not in touch with any devs, let me know and I can try to organize get you in touch with the right people.如果您没有与任何开发人员取得联系，请告诉我，我可以尝试安排您与合适的人取得联系。

Hello, the previous software engineer resigned, so no one took over this new target submitted in April 2024, I haven't been in touch with the developers on your side for the time being, can you help arrange for the relevant developers to contact me, I entered your INAV discord community.

sensei-hacker commented:
Can you message me or talk on the Discord so we know who you are?  I'm Sensei on the Discord as well.

sensei-hacker committed:
Change TARGET_BOARD_IDENTIFIER to 'BLF4' and update features

sensei-hacker committed:
Clean up timer definitions in target.c

Removed commented-out timer definitions and adjusted the order of active timer definitions.

sensei-hacker committed:
Change TARGET_BOARD_IDENTIFIER and SPI bus settings
